db/schema.rb

```diff
@@ -30,6 +30,7 @@
     t.bigint "byte_size", null: false
     t.string "checksum", null: false
     t.datetime "created_at", null: false
+    t.string "service_name", null: false
```

Comment:
what is thiiiiiis

Reply:
https://github.com/umts/jobapps/blob/master/db/migrate/20210224185746_add_service_name_to_active_storage_blobs.active_storage.rb